projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/src/hooks/useSocialLogin.tsx

```diff
@@ -32,15 +43,32 @@ export default function useSocialLogin() {
 	const [ loginWindowRef, setLoginWindowRef ] = useState< Window >();
 
 	useEffect( () => {
-		wpcomRequest< UserInfo >( {
-			path: '/verbum/auth',
-			apiNamespace: 'wpcom/v2',
-		} ).then( res => {
-			userInfo.value = res;
+		if ( ! ( 'hasStorageAccess' in document ) ) {
+			canAccessCookies.value = canWeAccessCookies();
+		}
+
+		document.hasStorageAccess().then( hasAccess => {
+			canAccessCookies.value = hasAccess;
```

Comment:
This `canAccessCookies` variable is probably redundant. The Verbum iframe doesn't need cookie or storage access. Only the embedded REST proxy iframe needs it.

Therefore, the `isPublicAPIReady` signal should be enough. But the REST proxy iframe should check if it has cookie access before sending the `ready` message. It can do it by calling the `getWpApiCookie` function, i.e., reading the `wp_api` cookie from `document.cookie`, and verifying that it's not `undefined`.

The `wp_api` cookie can be an empty string (`' '`) in case the user is not logged in to WP.com. But it should be there, otherwise it means cookies are blocked.